lib/tasks/notifications.rake

```diff
@@ -0,0 +1,10 @@
+namespace :notifications do
+
+  desc 'Creates new system notification for all active users'
+  task :new_release => :environment do
```

Comment:
We will need a parameterized Rake task, because system notifications can be anything. Since parameters in Rake tasks are pretty rudimentary, one way to input a more detailed data into the task is to provide a single parameter, path to a `.yaml` file, and that file has the detailed data specified (e.g. title & message of notification). I mean, your call, maybe it's not neccesary.